03/app.js

```diff
@@ -42,17 +42,34 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
-
+    
+    if(typeof this[isbn] === 'undefined') {
+        return null;
+    } const titleLang = this[isbn]['title'][lang];
+    if(typeof titleLang === 'undefined') {
+        return null;
+    } else  {
+        return titleLang;
+    }
 }
 
-books.getTranlator = function(isbn, lang) {
 
+books.getTranslator = function(isbn, lang) {
+    
+    if(typeof this[isbn] === 'undefined') {
+        return null;
+    } const translator = this[isbn]['translator'][lang];
```

Comment:
Warto też sprawdzić czy `this[isbn]['translator']` istnieje.
Bo jeśli nie to ten kod spowoduje błąd i program przestanie działać :)